codeception.yml

```diff
@@ -128,7 +125,6 @@ include:
   - bundles/log-log-io-connector
   - bundles/log-slack-connector
   - bundles/mail-bcc
-  - bundles/mailjet-mail-connector
```

Comment:
Ist das richtig, dass `bundles/mailjet-mail-connector` entfernt wird?

Reply:
you are right, that was a mistake